go.mod

```diff
@@ -23,13 +23,6 @@ replace (
 	// packages (and old docker versions)
 	github.com/Sirupsen/logrus => github.com/sirupsen/logrus v1.7.0
```

Comment:
Maybe this also could be removed. At least removal is not causing compile time errors. @rhuss I believe you added this, was it causing compilation error with pack v0.14.0 before?

Reply:
Build working for me on Mac OS without this replace

Reply:
@matejvasek do you want to remove this also?

Reply:
Build also working for me.  The less requires the better!

Reply:
@zroubalik d'oh, I thought I did.

Reply:
No problem :) Fixed: https://github.com/boson-project/func/pull/258